docs/en/sql-reference/functions/string-functions.md

```diff
@@ -867,6 +867,34 @@ Result:
 
 Like `base58Decode` but returns an empty string in case of error.
 
+**Syntax**
+
+```sql
+tryBase58Decode(encoded)
+```
+
+**Parameters**
+
+- `encoded`: [String](../../sql-reference/data-types/string.md) column or constant. If the string is not a valid Base58-encoded value, returns an empty string in case of error.
+
+**Returned value**
+
+- A string containing the decoded value of the argument.
+
+**Examples**
+
+Query:
+
+```sql
+SELECT tryBase58Decode('3dc8KtHrwM') as res;
```

Comment:
It would be useful to include an invalid base58 value as example, e.g.

```sql
SELECT tryBase58Decode('3dc8KtHrwM') as res, tryBase58Decode('invalid') as res_invalid;
```